.github/workflows/tests.yml

```diff
@@ -344,3 +344,101 @@ jobs:
         with:
           name: ember-exam-execution-${{ matrix.browser }}
           path: ./app/assets/javascripts/discourse/test-execution-*.json
+
+  theme_frontend_tests:
```

Comment:
Any way we can combine this into the main matrix? Copy/pasting will make this workflow more difficult to maintain into the future.

I suspect the reason for this is that theme tests only work in `RAILS_ENV=development` ? I ran into this problem when [setting up our internal CI ](https://github.com/discourse-org/all-the-custom-plugins/blob/main/concourse/run_tests.sh#L50-L56) to run theme tests. We can do all the preparation with `RAILS_ENV=test`, and then when we actually run the theme:qunit rake task we can set

```
DISCOURSE_DEV_DB=discourse_test
```

Reply:
> We can do all the preparation with RAILS_ENV=test, and then when we actually run the theme:qunit rake task we can set

I’ll need to see if running rails server with the test environment is actually possible. Technically it should be but will have to test it out.

Reply:
It works in our internal CI :+1:

Reply:
This turned out to be a really easy change. Thank you @davidtaylorhq :+1: